From 59a19def80d531c30b3c4a1d8f18c3c58985fee2 Mon Sep 17 00:00:00 2001
From: Abderrahim Kitouni <akitouni@gnome.org>
Date: Fri, 2 Dec 2022 15:47:09 +0100
Subject: [PATCH] components/dbus-base.bst: disable libaudit

This was added automatically because it was enabled in systemd
---
 elements/components/dbus-base.bst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/elements/components/dbus-base.bst b/elements/components/dbus-base.bst
index ab834d9bf..55a955707 100644
--- a/elements/components/dbus-base.bst
+++ b/elements/components/dbus-base.bst
@@ -13,6 +13,7 @@ variables:
     runstatedir=/run
     --with-system-socket=/run/dbus/system_bus_socket
     --enable-user-session
+    --disable-libaudit
 
 config:
   install-commands:
-- 
2.38.1

